### DISCUSSION TITLE
扫码登录中安装dotnet出现错误，我该怎么办？求大佬指点

### DISCUSSION THREAD

**Discussion by xiaonanboy** [RayWangQvQ/BiliBiliToolPro#580] [General]
出现这个东西，安装不了dotnet，是不是我的玩客云不支持。

Error relocating /root/.dotnet/dotnet: __time64: symbol not found
Error relocating /root/.dotnet/dotnet: __gmtime64: symbol not found
Error relocating /root/.dotnet/dotnet: __fstat_time64: symbol not found
Error relocating /root/.dotnet/dotnet: __dlsym_time64: symbol not found
Error relocating /root/.dotnet/dotnet: __fstatat_time64: symbol not found
安装失败，没办法了，毁灭吧，自己解决吧：https://learn.microsoft.com/zh-cn/dotnet/core/install/linux-alpine

**Comment by yehuzhucha** [comment]
同问

**Comment by leige635539766** [comment]
大家解决了吗？是因为玩客云是arm32的？

**Comment by RayWangQvQ** [comment]
稍后优化基于青龙的部署

**Comment by xulifu0415** [comment]
我的也是安装失败了

**Comment by RayWangQvQ** [comment]
请使用最新的dev先行版尝试

**Comment by fizzyob** [comment]
各脚本运行都提示：
/ql/config/config.sh: line 406: 2024-03-14: command not found
/ql/config/config.sh: line 408: 2024-03-18: command not found
bili_task_base.sh: line 76: /ql/shell/env.sh: No such file or directory

青龙版本2.11.3
